app/presenters/content_item_presenter.rb

```diff
@@ -44,14 +44,28 @@ def as_json(options = nil)
 
   def links
     return item.expanded_links unless document_type_expanded?
-    Rails.application.statsd.time('public_content_item_presenter.links') do
-      item.linked_items.each_with_object({}) do |(link_type, linked_items), items|
-        items[link_type] = linked_items.map { |i| present_linked_item(i) }
-      end
+    passthrough.merge(available_translations: available_translations)
+  end
+
+  def passthrough
+    item.links.each_with_object({}) do |(link_type, passthrough_hash), result|
+      passthrough, ids = passthrough_hash.partition { |type| type.is_a?(Hash) }
+      hashes = passthrough.map { |attributes| present(ContentItem.new(attributes)) }
+      content_items = ids.map { |id| present(ContentItem.find_by(content_id: id)) }
```

Comment:
Not sure if intended or not as ActiveRecord & Mongoid differ here but `find_by` will raise an exception if an item isn't found here - I'm not sure how likely that scenario is.
Also could be more performant and easier to read if this was `content_items = ContentItem.where(:id.in => ids).map ... `